packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -140,91 +191,129 @@ function NetworksPanel({
     [networkSearched]
   )
 
-  const networksToShowWithChainTypeInfo = useMemo(() => {
+  const networksToShow = useMemo(() => {
     const _networkSearched = debouncedNetworkSearched.trim().toLowerCase()
 
     if (_networkSearched) {
-      return networks.filter(network => {
-        const networkName = getBridgeUiConfigForChain(
-          network.chainId
-        ).network.name.toLowerCase()
+      return chainIds.filter(chainId => {
+        const networkName =
+          getBridgeUiConfigForChain(chainId).network.name.toLowerCase()
         return networkName.includes(_networkSearched)
       })
     }
 
-    const coreNetworks = networks.filter(network => network.type === 'core')
-    const orbitNetworks = networks.filter(network => network.type === 'orbit')
+    const coreNetworks = chainIds.filter(
+      chainId => !isNetwork(chainId).isOrbitChain
+    )
+    const orbitNetworks = chainIds.filter(
+      chainId => isNetwork(chainId).isOrbitChain
+    )
+
+    return {
+      core: coreNetworks,
+      orbit: orbitNetworks
+    }
```

Comment:
Do we ever use core and orbit separately? Otherwise we can just merge it here directly

Reply:
yes we do, it's clearer this way